cves/CVE-2014-9604.yml

```diff
@@ -240,8 +241,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: The file is located in the avcodec folder.
```

Comment:
It was explicitly stated that "Note: a filepath is NOT a subsystem - that's too granular." so your reasoning can't just be that its in the folder. I'm not saying the this isn't in the avcodec subsystem, but you'll probably need another reason for it